jsonpb/jsonpb.go

```diff
@@ -118,6 +118,9 @@ type JSONPBUnmarshaler interface {
 
 // Marshal marshals a protocol buffer into JSON.
 func (m *Marshaler) Marshal(out io.Writer, pb proto.Message) error {
+	if pb == nil || reflect.ValueOf(pb).IsNil() {
```

Comment:
This assumes that pb is a pointer, which is not always true.

Reply:
Ah yes.  Switched to checking for pointer only.  I'm guessing I shouldn't worry about nil maps, slices, etc.